data/pages.json

```diff
@@ -108,5 +108,10 @@
         "description": "Imposter 404",
         "imageUrl": "https://raw.githubusercontent.com/tsparticles/404-templates/main/__screenshots/imposter.png?raw=true",
         "url": "imposter/404.html"
+    },
+    {
+        "description":"Matrix 404",
+        "imageUrl":"https://raw.githubusercontent.com/RajSpeed/404-templates/matrix/__screenshots/matrix.png?raw=true",
```

Comment:
This URL is not correct, please use this repository as a source for images

Reply:
Done😀